rllib/agents/dqn/dqn.py

```diff
@@ -34,37 +35,30 @@
     # N-step Q learning
     "n_step": 1,
 
-    # === Exploration ===
-    # Max num timesteps for annealing schedules. Exploration is annealed from
-    # 1.0 to exploration_fraction over this number of timesteps scaled by
-    # exploration_fraction
-    "schedule_max_timesteps": 100000,
+    # === Exploration Settings ===
+    "exploration": {
+        # The Exploration class to use.
+        "type": "ray.rllib.utils.explorations.epsilon_greedy.EpsilonGreedy",
+        # Config for the Exploration class' c'tor:
+        "initial_epsilon": 1.0,  # Initial epsilon value.
+        "final_epsilon": 0.02,  # Final epsilon value.
```

Comment:
These two comments are redundant with the variable and should be removed.

Reply:
done